kubernetes/resource_kubernetes_namespace.go

```diff
@@ -88,6 +114,15 @@ func resourceKubernetesNamespaceUpdate(d *schema.ResourceData, meta interface{})
 		return fmt.Errorf("Failed to marshal update operations: %s", err)
 	}
 
+	if d.HasChange("spec") {
```

Comment:
This way of constructing the patch operation by reusing the expand.. helper won't work. It seems that patch operations need to be constructed at the primitive attribute level, for each attribute in turn.

Please have a look at an existing implementation like `func patchPodSpec(...)` for inspiration on how to implement here.

Reply:
I had to do something like this recently too. You can check out my example if you want. https://github.com/dak1n1/terraform-provider-kubernetes/blob/c245d421be2e2070e8cb61a03a6cc4f026d5c07c/kubernetes/resource_kubernetes_persistent_volume_claim.go#L197

You'll also want to move the `ops.MarshalJSON()` a few lines down to make sure you don't marshall it until all the operations have been appended. Otherwise the Update is performed without your changes.

Reply:
ive changed it to update according to the comments above, thanks for that. the body of the request seem to be fine now but the update doesnt fail but doesnt really work...

any advice?